docs/config.md

```diff
@@ -81,6 +84,8 @@ or alternatively, using a dict to map the prefix path to the format name:
 "notebooks/" = "ipynb"
 "scripts/" = "py:percent"
 ```
+Note that if you are using a `pyproject.toml` file with this dict format, you should make sure the table header is instead `[tool.jupytext.formats]`
```

Comment:
Thanks for documenting this. By the way I was thinking that it would be helpful to give a small example `pyproject.toml` file, say e.g.
```
[tool.jupytext]
formats = "ipynb,py:percent"
```
Where do you think this example would be best located in the documentation? Maybe right after the sample `jupytext.tom` file with the same configuration?